### PR TITLE
W-10862773 first pass

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/mq-queues.adoc [hunk 1]
```diff
@@ -342,26 +342,39 @@ The time it takes to delete or purge a queue is approximately one minute. During
 
 For applications where the order of messages must be strictly preserved and enforced, Anypoint MQ provides first in, first out (FIFO) processing to enable message ordering. With FIFO queues, Anypoint MQ ensures that the order in which messages are placed in a queue is the same order messages are retrieved.
 
-FIFO queues have the following benefits and restrictions:
+Benefits of FIFO queues::
++
+* Support delivery delay on all messages in the FIFO queue.
++
+If you change the delivery delay for a FIFO queue, the setting change is retroactive to all messages in the queue.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-queues.adoc]: (reworded)Change in delivery delay setting for a FIFO queue, gets applied retroactively to all messages in the queue.
 (on next line?) Anypoint MQ doesn't support delayed delivery for individual messages in a FIFO queue.

File: modules/ROOT/pages/mq-queues.adoc [hunk 2]
```diff
@@ -342,26 +342,39 @@ The time it takes to delete or purge a queue is approximately one minute. During
...
+Anypoint MQ doesn't support delayed delivery for individual messages in a FIFO queue. 
+* Support multiple <<FIFO Queues and Message Groups,message groups>>. 
+* Provide encryption.
++
+Encrypting a FIFO queue does not affect the order or the contents of messages consumed.
+* Can be used as a DLQ; however, it is not recommended. See the limitations.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-queues.adoc]: I think we should add a separate bullet for DeadLetter Queues and add details wrt standard and FIFO queues?

If not, here is what I think we should add - 

FIFO can be configured with Dead Letter Queue (DLQ). DLQ of a FIFO queue must also be a FIFO queue.
Strict ordering can't be guaranteed while using FIFO with DLQs.

File: modules/ROOT/pages/mq-queues.adoc [hunk 3]
```diff
@@ -342,26 +342,39 @@ The time it takes to delete or purge a queue is approximately one minute. During
...
++
+If you assign a DLQ to a FIFO queue, the DLQ must also be a FIFO queue.
 
-* Are limited to 300 TPS (transactions per second) 
+Limitations of FIFO queues::
 +
-However, if you batch 10 messages per each read and write operation (maximum) using the API, FIFO queues can support up to 3,000 TPS.
+* Are limited to 300 TPS (transactions per second).
++
+However, if you batch ten messages per read and write operation (maximum) using the API, FIFO queues can support up to 3,000 TPS.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-queues.adoc:363]: this should be extension of previous point about 300TPS.

File: modules/ROOT/pages/mq-queues.adoc [hunk 4]
```diff
@@ -342,26 +342,39 @@ The time it takes to delete or purge a queue is approximately one minute. During
...
 +
 //Shared TPS failed
 include::partial$mq-faq.adoc[tag=failedTPS]
-* Support delivery delay on all messages in the FIFO queue
+
+* Can't guarantee that messages in a message group are processed in order when used with a DLQ.
```
Comment (kimayap03) [inline modules/ROOT/pages/mq-queues.adoc]: Strict ordering within message group can't be guaranteed while using FIFO with DLQs.

Comment (kimayap03) [reply modules/ROOT/pages/mq-queues.adoc]: When a message is sent to a DLQ, it is deleted from the FIFO queue.
Anypoint MQ processes subsequent messages from FIFO upon request.
Hence Do not use FIFO queue with a DLQ if strict order needs to be preserved.

**Review by kimayap03 — APPROVED** [review]
LGTM

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**